src/processors/encounter/encounter-form-processor.ts

```diff
@@ -202,6 +203,26 @@ export class EncounterFormProcessor extends FormProcessor {
           critical: true,
         });
       }
+      // handle appointments
+      try {
+        const {appointments: myAppointments} = context
+        const appointmentsResponse = await Promise.all(addFulfillingEncounters(abortController, appointments, savedEncounter.uuid));
+        if (appointmentsResponse?.length) {
+          showSnackbar({
+            title: translateFn('appointmentsSaved', 'Appointment(s) saved successfully'),
```

Comment:
Does `translateFrom` handle pluralization? If not, I think we could just default it to plural:

```suggestion
            title: translateFn('appointmentsSaved', 'Appointments saved successfully'),
```